lib/routes/shu/gs.ts

```diff
@@ -0,0 +1,93 @@
+import { Route } from '@/types';
+import cache from '@/utils/cache';
+import got from '@/utils/got';
+import { load } from 'cheerio'; // cheerio@1.0.0
+import { parseDate } from '@/utils/parse-date';
+import timezone from '@/utils/timezone';
+
+const notice_type = {
+    zhxw: { title: '上海大学研究生院-综合新闻', url: 'https://gs.shu.edu.cn/xwlb/zh.htm' }, // 综合新闻
+    pygl: { title: '上海大学研究生院-培养管理', url: 'https://gs.shu.edu.cn/xwlb/py.htm' }, // local //BUG error: Request https://gs1.shu.edu.cn:8080/py/KCBInfo.asp fail: TypeError: fetch failed
+    gjjl: { title: '上海大学研究生院-国际交流', url: 'https://gs.shu.edu.cn/xwlb/gjjl.htm' },
+};
+
+export const route: Route = {
+    path: '/gs/:type?',
+    categories: ['university'],
+    example: '/shu/gs/zhxw',
+    parameters: { type: '分类，默认为学术公告' },
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    radar: [
+        {
+            source: ['gs.shu.edu.cn/'],
+            target: '/gs',
+        },
+    ],
+    name: '研究生院',
+    maintainers: ['GhhG123'],
+    handler,
+    url: 'gs.shu.edu.cn/',
+    description: `| 综合新闻 | 培养管理 | 国际交流 |
+  | -------- | --------- | --------- |
+  | zhxw     | pygl      | gjjl      |`,
+};
+
+async function handler(ctx) {
+    const type = ctx.req.param('type') ?? 'zhxw';
+    const rootUrl = 'https://gs.shu.edu.cn';
+
+    // 发起 HTTP GET 请求
+    const response = await got({
```

Comment:
Use either https://github.com/DIYgod/RSSHub/blob/8b48453ed55165a9e2adbda0705bb8bf6accd1e1/lib/routes/shu/gs.ts#L47-L55 or https://github.com/DIYgod/RSSHub/blob/8b48453ed55165a9e2adbda0705bb8bf6accd1e1/lib/routes/shu/gs.ts#L77 but not both style